frontend/lib/norent/footer.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { NorentRoutes as Routes } from "./routes";
+import { Link } from "react-router-dom";
+
+const MAILCHIMP_URL =
+  "https://nyc.us13.list-manage.com/subscribe/post?u=d4f5d1addd4357eb77c3f8a99&amp;id=588f6c6ef4";
+
+const EmailSignupForm = () => (
+  <form
+    action={MAILCHIMP_URL}
+    className="email-form is-horizontal-center"
+    method="post"
+    target="_blank"
+  >
+    <div className="mc-field-group">
+      <div className="control is-expanded">
+        <input
+          type="email"
+          name="EMAIL"
+          className="required email input"
+          id="mce-EMAIL"
+          placeholder="Enter your email"
+        />
+      </div>
```

Comment:
So this needs a `<label htmlFor="some-id">` associated with it.  I recommend giving it the `jf-sr-only` class so it's only visible to assistive tech... You'll want to give the input element a unique `id` to associate the label w/ it but since it's going to only appear once per page, I think you can just hard-code it, although I suppose it would be best practice to have it passed in as a prop (that way it keeps things explicit, and it's not a big deal for usability since the component will be used sparingly).

FYI the forms book _does_ say that putting labels as `placeholder` text is not great for various reasons, so we should definitely use it sparingly if at all, but that's more feedback to pass on to Tahnee than anything for this PR.

Reply:
Er, it's possible that `aria-label` might work here instead of a whole `<label>` doohickey too, but I'm not sure.  The `<label>` version has the most support across browser+AT combinations, I think...

Reply:
Ah sounds good! Also, did you know that [Bulma](https://bulma.io/documentation/modifiers/helpers/) has an `.is-sr-only` helper class now? I think it was added in a more recent release, perhaps?